src/endpoints/getResults.ts

```diff
@@ -6,7 +6,7 @@ import { popSlashSource } from '../utils/parsing'
 import { HLTVConfig } from '../config'
 import { fetchPage, toArray, getMatchFormatAndMap } from '../utils/mappers'
 
-export const getResults = (config: HLTVConfig) => async ({ pages = 1 } = {}): Promise<
+export const getResults = (config: HLTVConfig) => async ({ pages = 1, teamID = 0 } = {}): Promise<
```

Comment:
Why is the default to `= 0` needed here?

Reply:
No, not needed. Removed it